keyserver/listener.go

```diff
@@ -32,7 +33,6 @@ func (server *ConiksServer) listenForRequests(ln *net.TCPListener, handler func(
 			log.Printf("accept client: %s", err)
 			continue
 		}
-		conn = tls.Server(conn, server.tlsConfig)
```

Comment:
> But, out of curiosity, why do you prefer that?

Since I also handle an unix socket connection here (https://github.com/coniks-sys/coniks-go/blob/63ef04220a778a4707329a701625dbdd16470684/keyserver/listener.go#L40-L42).